client/common/sass/components/_media.sass

```diff
@@ -1,16 +1,19 @@
 .inline-media
-	&.full-width
-		margin-inline: 0
-
-	&.left
-		float: left
-		margin-bottom: 1em
-		margin-right: 1em
-
-	&.right
-		float: right
-		margin-bottom: 1em
-		margin-left: 1em
+	width: 100%
+	margin-inline: 0
+
+	+tablet-up
+		&.left
+			margin-bottom: 1em
+			margin-right: 1em
+			float: left
+			max-width: 60%
```

Comment:
This value is slightly randomly decided. I just felt the media should take up slightly more space than the text even when the text wraps. But if people feel strongly about keep this to 50% so that they take up almost equal space, or even increasing it to make the media take up more space than now, let me know.